app/views/talents/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="container">
+  
+  <div class="row mt-20 mb-20">
+    <div class="col-xs-6 col-sm-6 col-md-6 col-lg-6">
+      <h2>New Talent</h2>
+      <!-- Button trigger modal -->
+    </div>
+    <div class="col-xs-6 col-sm-6 col-md-6 col-lg-6">
+      <button type="button" class="btn btn-primary float-right" data-toggle="modal" data-target="#batchModal">
+        Upload Talents
+      </button>
+    </div>
+  </div>
+  <hr>
+  <%= render partial: 'form', locals: { :talent => @talent } %>
```

Comment:
`partial` & `locals` keywords are no longer needed, you can write the above like this: 

```ruby
render 'form', talent: @talent
```